tests/integration/models/reddit/test_live.py

```diff
@@ -42,3 +46,115 @@ def test_updates(self, _):
             for update in thread.updates(limit=None):
                 assert update.thread == thread
         assert update.body.startswith('Small change:')
+
+
+class TestLiveContributorRelationship(IntegrationTest):
+    @contextmanager
+    @mock.patch('time.sleep', return_value=None)
+    def ensure_no_invite(self, thread, redditor, _):
+        thread.contributor.remove_invite(redditor)
+        yield
+        thread.contributor.remove_invite(redditor)
```

Comment:
I thought the same thing while I wrote the test. 

> My approach typically when generating cassettes is to manually set the state to what I expect it to be prior to running the test for the first time. While this means the cassettes cannot be easily generated in bulk, it means the test code and the cassettes are simpler.

Thanks for the tips! I'll try this.